d2/src/main/java/com/linkedin/d2/xds/XdsClient.java

```diff
@@ -59,19 +63,22 @@ interface D2URIMapResourceWatcher extends ResourceWatcher
 
   }
 
+  interface D2URICollectionResourceWatcher extends ResourceWatcher
+  {
+    void onChanged(D2URICollectionUpdate update);
+  }
+
   interface ResourceUpdate
   {
   }
 
 
   static final class NodeUpdate implements ResourceUpdate
   {
-    String _version;
```

Comment:
I wanted to keep it consistent in case we needed it. In the end, we don't really use it and in the case of glob collections, it actually doesn't make much sense since there is no version for the update, only the versions for each URI